### PR TITLE
#1-4Task. ivan.lozovskij

### PR DESCRIPTION
**Pull request by Lozovskij**
https://travis-ci.org/github/Lozovskij/js-assignments/builds/705084597

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -114,9 +117,15 @@ function removeLeadingAndTrailingWhitespaces(value) {
  *   'cat', 3 => 'catcatcat'
  */
 function repeatString(value, count) {
-    throw new Error('Not implemented');
+    if (count == 0) { return ''; }
+    var res = value;
```
Comment (FedorovD) [inline task/01-strings-tasks.js:121]: `var` is deprecated. It has no sense to use for browsers code even currently.

File: task/01-strings-tasks.js [hunk 2]
```diff
@@ -201,8 +216,26 @@ function extractEmails(str) {
...
+    if (h == 2) {return "";}
+    var middleLayer = '│' + repeatString(' ', w - 2) + '│\n';
+    return repeatString(middleLayer, h - 2);
 }
+function getBottom(w) {
+    return '└' + repeatString('─', w - 2) + '┘\n';
+}
+console.log(getRectangleString(2,2));
+
+
```
Comment (FedorovD) [inline task/01-strings-tasks.js:238]: do not create unnecessary functions, solve the task inside the existing one

File: task/01-strings-tasks.js [hunk 3]
```diff
@@ -221,7 +254,15 @@ function getRectangleString(width, height) {
...
+    var str1 = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+    var str2 = 'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
+    var res = '';
+    for (var i = 0; i < str.length; i++) {
+        var ind = str2.indexOf(str[i]);
+        if (ind == -1) { res += str[i];}
+        else { res += str1[ind]; }
+    }
+    return res;
 }
```
Comment (FedorovD) [inline task/01-strings-tasks.js:266]: `var` is deprecated. It has no sense to use for browsers code even currently.

**Comment by FedorovD** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by Lozovskij** [comment]
https://travis-ci.org/github/Lozovskij/js-assignments/builds/706845042

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**